packages/offline-payment-integration/tsconfig.lib.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
+  "include": [
+    "src/**/*.ts",
+    "src/**/*.tsx"
+  ]
+}
```

Comment:
I must miss this one before. Think we discussed removing `lib.json` before. 
The rest look good to me.

Reply:
I can't recall correctly the discussion, but can remove this since don't think it's used.